.github/workflows/tests_linux.yml

```diff
@@ -67,9 +67,9 @@ jobs:
             lcov --remove coverage.info '/usr/*' --output-file coverage.info
 
       - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v2.1.0
+        uses: codecov/codecov-action@v2
         with:
-          file: ./BuildCov/coverage.info
```

Comment:
How come these were changed?

Reply:
Hi @antalszava! Somehow Codecov raised errors today. I found that the Codecov document does not have `file: ` option anymore, and just using `@v2` is recommended.

Reply:
Could you link a related source? Curious if we'll see this happening elsewhere too :thinking:

Reply:
Here is the link: https://github.com/codecov/codecov-action#usage